docs/standard/security/fips-compliance.md

```diff
@@ -0,0 +1,28 @@
+---
+title: .NET Core FIPS compliance
+description: Explains .NET Core FIPS compliance
+ms.date: 11/20/2019
+---
+
+# .NET Core FIPS compliance
+
+.NET Core:
+
+* Does **not** implement any cryptographic primitives.
+* Passes cryptographic primitives calls through to the standard modules the underlying operating system provides.
+* Does **not** enforce FIPS compliance in .NET Core apps.
+* Does **not** stop developers from using non-FIPS compliant algorithms, even on systems configured for FIPS compliance.
+
+It's the responsibility of the system administrator to configure FIPS compliance for an operating system.
+
+If code is written for a FIPS-compliant environment, it's the responsibility of the developer to:
+
+* Ensure that non-compliant FIPS algorithms are not used.
+* FIPS compliance is met.
```

Comment:
Not in parallel grammatically to previous bullet.  Suggest:  "... the developer to ensure that: non-compliant FIPS algorithms are not used, FIPS compliance is met."